BackofficeBundle/Resources/public/coffee/tab/tabViewFormLoader.coffee

```diff
@@ -0,0 +1,28 @@
+tabViewFormLoad = (options) ->
+  tabViewClass = appConfigurationView.getConfiguration(options.entityType, 'showTab')
+  elementTabViewClass = appConfigurationView.getConfiguration(options.entityType, 'editEntityTab')
+
+  tabView = new tabViewClass(options)
+  panels = getPanelsLink(options.element.get('links'))
+
+  for panel, i in panels
+    do (panel, i) ->
+      $.ajax
+        url: panel.link
+        method: "GET"
+        success: (response) ->
+          view = new elementTabViewClass(
```

Comment:
we should be able to have a different view for each tab
@nicolasThal @itkg-alavieille